src/main/java/seedu/address/logic/commands/MarkCustomerOrderCommand.java

```diff
@@ -0,0 +1,95 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.order.CustomerOrder;
+import seedu.address.model.order.CustomerOrderList;
+import seedu.address.model.order.OrderStatus;
+import seedu.address.model.product.Ingredient;
+import seedu.address.model.product.Pastry;
+import seedu.address.model.product.Inventory;
+import seedu.address.model.product.Product;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MarkCustomerOrderCommand extends Command {
```

Comment:
It seems abit messy here, may be you can try tidy the codes up for the next iteration.